corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -193,6 +194,13 @@ <h3 data-bind="text: $root.activeCaseType()" style="display: inline-block;"></h3
           {% trans "Delete Case Type" %}
         </a>
       {% endif %}
+      {% if request.couch_user.can_edit_data %}
+        <a class="btn btn-default" id="export-case-type-data-dict"
```

Comment:
This ID will allow setting up analytics without additional code, right? Redundant buttons can be useful, but I think it's good to gather evidence of how often they end up getting used.

Reply:
Hey @orangejenny 
ya that is my intent was with that ID.

> Redundant buttons can be useful

Not sure I understand here. Are you suggesting we should not put IDs unless needed or use anchor instead of buttons?

Reply:
No, by "redundant" I meant that this is an additional way to navigate to the export page, beyond the existing HQ menu.

Reply:
ah that, Yes.
Honestly, I don't think this would get used as much as or even close to "Export case data" on Data tab. It is just a convenient link from Data dictionary.
Just pinging @esoergel here, if he has anything to add to why he thought we should add this. Though this did change from how it was initially thought about.

Reply:
Are you referring to the last line in [this CEP](https://docs.google.com/document/d/1rpCFUXOAJAFBCmh8GMfivShOHfKjskdWPdE2ZPFkqP0/edit)?

> Is there a place for a “default” export per case type?  Like, we can make an export config based on the current version of the DD - maybe we should do this on the fly and give users an unconfigurable export per case type that is always up-to-date and ready to download.

I was imagining a default export config for each case type, available via the "export case data" tab, so the user doesn't need to worry about creating or updating configs unless they want to do something specific.  Here's a mock up of what I mean (the first one)

![image](https://github.com/dimagi/commcare-hq/assets/2367539/b97f579d-1e47-4f9f-8da3-7619f2f8ebae)

Though we'd probably want to do more to visually distinguish it

Reply:
Thanks @esoergel 

I don't have the context, but the way it was [specced out](https://docs.google.com/document/d/1L45d4FEdepIjHPWNvEa2e5gbophp9ZavOlt7D1JpkKU/edit#heading=h.5uixuydcq2ij) was like the way you see it implemented in this PR.

> I was imagining a default export config for each case type

I believe if we add a default export for each case type on exports, it could get really crowded specially for projects with multiple case types? So, getting these added by the user seems like a better approach.

>  Right now when you go to "Export Case Data" on a new domain, you just get a page that says "No Exports to display," then you have to generate a new export, which seems needless when the defaults are probably fine most of the time anyways

Right. Agreed that defaults are almost always okay and users probably don't even change anything. We are saving users three clicks ("Add Export", choosing case type, "save") here then.

> Here's a mock up of what I mean. The first one in the list is always there, the second required the user to go through the config screen to set up an export.

I believe even the first one would have an active schema added to it and is not just a placeholder.

> Furthermore, if you add case properties to your app, you have to either create new export configs or manually update old ones. Using a default config that is built on the fly from the DD makes the basic "download my data" use case easier.

Adding a config to export to keep it up to date with new changes to the project via data dictionary seems like what will do what you want & I like that.

I do think its hard for HQ to decide reliably which case types would be valuable to export for the project out of the ones present and hence add a default export for them along with a schema.

Let me know if
1. you think we should reconsider how we are doing this?
and/or add a config to exports to keep them up to date with data dictionary automatically.
2. not add a button to the data dictionary like in this PR since it wont really be used much? I don't totally disagree with the concern Jenny had.

Reply:
> not add a button to the data dictionary like in this PR since it wont really be used much

I agree with this - if it's just another way to get to the same screen and doesn't skip the configuration, I don't think it's super valuable.

> it could get really crowded specially for projects with multiple case types

That might be true - I don't think I have great intuition there.  That might be a UX problem then - since the only action for a default export is "download", maybe these don't belong in the table with user-configured exports.  Instead it could be a single button per case type above the table, or a single-select dropdown on the "Download Case Data Export" or something else altogether.

> We are saving users three clicks

This is me being opinionated here, but I do like the [Don't Make Me Think](https://en.wikipedia.org/wiki/Don't_Make_Me_Think) rationale - it's easy for an experienced user to click through creating an export, but new users probably find the config page intimidating.

> add a config to exports to keep them up to date with data dictionary automatically

I do think both points are salient:
* Easily download all data for a case type without creating a stored config
* Have an export that gets new case properties automatically

Default exports would address both of them, but if you don't like that idea and want to focus on the second point, that also sounds potentially useful.  I could see that idea being an extension of the "Select all" button.

At the end of the day, I'm throwing out ideas that I think are useful, but since you're actually making this into a reality I fully support you owning the direction it takes, and I'm happy to weigh in if helpful.  Might be worth putting together a product doc for review - I'm sure product and delivery folks would have more informed opinions than either of us on what would be useful.